### PR TITLE
fix(lnurlpay): correct close for popup/prompt

### PR DESCRIPTION
**Pull request by escapedcat**
### Describe the changes you have made in this PR

Align close actions for all actions. Those will not be reached by the modal because it is closing itself on success.

### Link this PR to an issue

#1216 

### Type of change (Remove other not matching type)

- `refactor`

### How has this been tested?
Manually tested with fiatjef and starblocks

### Checklist

- [x] My code follows the style guidelines of this project and performed a self-review of my own code
- [x] New and existing tests pass locally with my changes
- [x] I checked if I need to make corresponding changes to the documentation (and made those changes if needed)

### CODE REVIEW

**Comment by escapedcat** [comment]
> the success view in which context?
> the prompt must reply that the response is sent back to the `webln.lnurl` call for example.

Ah ok, hm, need to test this. Will switch the PR to draft.

**Comment by bumi** [comment]
is this mergeable

File: src/app/screens/LNURLPay/index.tsx [hunk 1]
```diff
@@ -205,9 +205,10 @@ function LNURLPay() {
   function close(e: MouseEvent) {
     e.preventDefault();
     if (navState.isPrompt) {
-      msg.reply(payment);
-    } else {
+      msg.reply(payment); // will thsi ever be reached? according to the comment in line 181 maybe not?
       window.close();
```
Comment (bumi) [inline src/app/screens/LNURLPay/index.tsx:209]: this must not be both, either msg.reply or window.close

**Comment by bumi** [comment]
what's up with this one?

File: src/app/screens/LNURLPay/index.tsx [hunk 2]
```diff
@@ -205,12 +202,9 @@ function LNURLPay() {
   }
 
   function close(e: MouseEvent) {
+    // will never be reached via prompt
     e.preventDefault();
-    if (navState.isPrompt) {
-      msg.reply(payment);
```
Comment (escapedcat) [inline src/app/screens/LNURLPay/index.tsx:210]: We already do this above:
```ts
  if (navState.isPrompt) {
    msg.reply(paymentResponse);
  }
```

**Review by bumi — APPROVED** [review]
utACK

**Pull request merged**